config/types/schema.go

```diff
@@ -151,7 +149,7 @@ type PasswdGroup struct {
 type PasswdUser struct {
 	Create            *Usercreate        `json:"create,omitempty"`
 	Gecos             string             `json:"gecos,omitempty"`
-	Groups            []Group            `json:"groups,omitempty"`
```

Comment:
Is it this one? https://github.com/idubinskiy/schematyper There are quite a few, I'll add this to the development doc